python-sdk/tests/databases/test_bigquery.py

```diff
@@ -139,3 +140,21 @@ def mock_get_dataset(dataset_id):
 
     db = BigqueryDatabase(table=source_table, conn_id="test_conn")
     assert db.populate_table_metadata(input_table) == returned_table
+
+
+@mock.patch("astro.databases.google.bigquery.BigqueryDatabase.hook")
+def test_get_project_id_raise_exception(mock_hook):
```

Comment:
This test is moved from integration tests to unit test right?

Reply:
yes